index.js

```diff
@@ -4,6 +4,12 @@ module.exports = function buildAirbnbPreset(context, options) {
   var preset; // eslint-disable-line no-var
   if (options && options.modules === false) {
     preset = require('babel-preset-es2015').buildPreset(null, { modules: false });
+  } else if (options && options.node === true) {
```

Comment:
Hmmmm ok so relevant to #13 I'm guessing we should also leave the browser config options to the user but have defaults?